crates/wasi-preview1-component-adapter/wit/deps.lock

```diff
@@ -0,0 +1,41 @@
+[clocks]
```

Comment:
Could this be left out until CI is added to verify it's right?

Reply:
Yes, this should be left out until we actually setup wit-deps to point at the standards repos and check things properly in CI.